activerecord/lib/active_record/errors.rb

```diff
@@ -173,9 +173,9 @@ class ValueTooLong < StatementInvalid
   class RangeError < StatementInvalid
   end
 
-  # Raised when number of bind variables in statement given to +:condition+ key
-  # (for example, when using {ActiveRecord::Base.find}[rdoc-ref:FinderMethods#find] method)
-  # does not match number of expected values supplied.
+  # Raised when the number of placeholders in an SQL fragment passed to
```

Comment:
Think it’s `a SQL fragment`

Reply:
It depends whether you pronounce SQL as an initialism or as "sequel", which makes it ambiguous and somewhat controversial. 😄

The precedent in the rest of the codebase is to use "an": https://github.com/rails/rails/search?q=%22an+SQL%22